eoscostarica/eosio-dashboard
# Feature/UI redesign footer #861

### Redesign Footer&Header

### What does this PR do?

- Resolve #861 

### Steps to test

1. make start
2. navigate to http://localhost:3000/ and see footer&header

#### CheckList
